src/Maestro/tests/FeedCleaner.Tests/FeedCleanerTests.cs

```diff
@@ -0,0 +1,322 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Maestro.AzureDevOps;
+using Maestro.Data;
+using Maestro.Data.Models;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Internal;
+using Microsoft.Extensions.DependencyInjection;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace FeedCleaner.Tests
+{
+    public class FeedCleanerTests : IDisposable
+    {
+        private readonly Lazy<BuildAssetRegistryContext> _context;
+        private readonly Mock<IHostingEnvironment> Env;
+        private readonly ServiceProvider Provider;
+        private readonly IServiceScope Scope;
+
+        private static readonly string someAccount = "someAccount";
```

Comment:
should the variable's name first letter be capitalized here?

Reply:
now I'm curious why only the account need to be global

Reply:
It doesn't need to be, just to avoid duplication all over.